### PR TITLE
Karpenter addon to allow setting ttlSecondsUntilExpired

### PR DESCRIPTION
**Pull request by youngjeong46**
*Issue #, if available:*
#518 

*Description of changes:*
Minor Karpenter addon change to allow setting ttlSecondsUntilExpired as requested per above issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/addons/karpenter/index.ts [hunk 1]
```diff
@@ -202,6 +210,7 @@ export class KarpenterAddOn extends HelmAddOn {
                         subnetSelector: subnetTags,
                         securityGroupSelector: sgTags,
                     },
+                    ttlSecondsUntilExpired: ttlSecondsUntilExpired,
```
Comment (shapirov103) [inline lib/addons/karpenter/index.ts:213]: did you validate that null setting does not break the install? I saw cases when null not permitted, however lack of key is OK.

**Review by shapirov103 — COMMENTED** [review]
Looks good, one comment/clarification before merging.

File: lib/addons/karpenter/index.ts (hunk 1 above)
Comment (youngjeong46) [reply lib/addons/karpenter/index.ts:213]: Yes - just validated and it works as intended (no value provided -> null -> no custom helm value provided)

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**